packages/protobuf-test/typescript/ts3_9_4/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "include": ["../../src/**/*.ts"],
+  "exclude": ["../../src/**/*.test.ts", "../../src/gen/**/*.ts"],
```

Comment:
As discussed, we can't really exclude `gen`. It is totally fair if we limit the tests to extra/ though. Those are pretty exemplary for what a user would generate. It will also speed up the tests considerably.